### ISSUE TITLE
Add i18n to centralize definition of text

### ISSUE THREAD

**Issue by devtobi** [it-at-m/refarch-templates#116]
**Is your feature request related to a problem? Please describe.**
Currently text is defined in the components itself. This is not good if we want to support i18n later on.
On top of that even when using i18n with only 1 language, we can better 

**Describe the solution you'd like**
Add i18n support using https://vue-i18n.intlify.dev/. 

**Describe alternatives you've considered**
---

**Additional context**
See https://github.com/devtobi/cv for an example

**Comment by devtobi** [comment]
Decided to close this as not planned, as we currently have not planned to support i18n for every application by default.

**Comment by eidottermihi** [comment]
As discussed today, if it is decided that an i18n mechansims based on vue-i18n should be provided, I'm happy to contribute a PR

**Comment by devtobi** [comment]
> As discussed today, if it is decided that an i18n mechansims based on vue-i18n should be provided, I'm happy to contribute a PR

Yes `vue-i18n` would be our library of choice. Feel free to provide a PR.

**Comment by darenegade** [comment]
> In an internal meeting we again talked about i18n and it is desired to support it by default now.

The question is how we support it. A few years ago, it was introduced as a default in the templates. As multilingualism is not necessary for internal LHM applications, it led to additional work without any benefit, so it was removed. With regard to the RefArch templates, we want to support several use cases and if multilingualism is not necessary in all cases, we can think about low-threshold support. For example, it may be sufficient to include hints and tips for quick implementation in the documentation or only implement them using an example in the templates so that not everyone is “forced” to do so.

However, the topic currently has a low priority for the RefArch team and will only be considered in more detail in the future. Code contributions are of course welcome, but please take into account the facts described above.
